CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Changelog
 
 ## Unreleased
+- Kafka topic config supports default topic names as keys ([#3282](https://github.com/getsentry/relay/pull/3282))
```

Comment:
```suggestion
- Kafka topic config supports default topic names as keys. ([#3282](https://github.com/getsentry/relay/pull/3282))
```

Reply:
Out of curiosity, is this normally generated by hand?

Reply:
Yes, usually done by hand, but many PRs without impact have no changelog entry (this one should)